packages/ui/src/components/carousel-item/carouselItem.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+  import { computed, getCurrentInstance, inject, onMounted, onUnmounted } from 'vue';
+  import { isDefined } from '@vueuse/core';
+  import type {
+    AcvCarouselItemProps,
+    AcvCarouselItemSlots
+  } from './carouselItem.ts';
+
+  import {
+    CAROUSEL_INJECTION_KEY
+  } from '@/components/carousel/carousel.ts';
+  import './carouselItem.css';
+
+  /**
+   * Short description of the CarouselItem component for the generated documentation...
+   *
+   * @component AcvCarouselItem
+   */
+  defineOptions({
+    name: 'AcvCarouselItem'
+  });
```

Comment:
do we need it?

Reply:
Some artifacts are auto-generated, I'll clean them up